### PR TITLE
Create database models

### PR DESCRIPTION
**Pull request by dangph-1996**
==> Tittle: init project

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/16347

### CODE REVIEW

**Comment by dangph-1996** [comment]
ready

File: app/models/football_club.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class FootballClub < ApplicationRecord
+  has_many :players, dependent: :destroy
+  has_many :rankings, dependent: :destroy
+  has_many :home_team, class_name: "Match", foreign_key: "home_team",
```
Comment (hungct-1121) [inline app/models/football_club.rb]: Match.name

File: app/models/match.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Match < ApplicationRecord
+  belongs_to :round
+  belongs to :home_team, class_name: "football_club"
```
Comment (hungct-1121) [inline app/models/match.rb]: FootballClub.name chứ nhỉ :-?

**Comment by dangph-1996** [comment]
ready

**Pull request merged**